components/header/index.tsx

```diff
@@ -32,7 +42,17 @@ export const Header = async ({ cart }: { cart: ReactNode }) => {
           </Link>
         </NavigationMenuLink>
 
-        <HeaderNav className="hidden xl:flex" />
+        {process.env.HYPA_MENU_CODE && process.env.BIGCOMMERCE_STORE_HASH ? (
+          <MegaMenuBuilder
+            className="hidden lg:block"
+            code={process.env.HYPA_MENU_CODE}
+            menuItems={menuItems}
+            customerGroupId={customerGroupId}
+            customerId={customerId}
+            storeHash={process.env.BIGCOMMERCE_STORE_HASH}
+            theme="desktop"
+          />
+        ) : null}
```

Comment:
I wonder if it would be better to return the original menu here instead of nothing?